internal/pkg/cli/task_delete.go

```diff
@@ -7,12 +7,17 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/aws/aws-sdk-go/aws"
 	awssession "github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/copilot-cli/internal/pkg/aws/ecr"
 	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
 	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/aws/copilot-cli/internal/pkg/deploy"
 	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/ecs"
```

Comment:
nice! I really appreciate how between the first and second revision the dependency on `aws-sdk-go/ecs` went away. As we work on https://github.com/aws/copilot-cli/issues/1852 it will be a lot easier to migrate to v2 if all sdk structs are contained within `internal/aws`